requirements.txt

```diff
@@ -4,4 +4,4 @@ torch>=1.8.0
 torchvision>=0.9.0
 tqdm>=4.20.0
 requests>=2.20.0
-pylocron>=0.1.3
+-e git+https://github.com/frgfm/Holocron.git#egg=pylocron
```

Comment:
would be safer to go for `pylocron>=0.2.0` and it would guarantee deps consistency between `requirements.txt` and `setup.py`